distributed/scheduler.py

```diff
@@ -1108,7 +1108,7 @@ class TaskState:
     #:     "processing" state and be sent for execution to another connected worker.
     loose_restrictions: bool
 
-    #: Whether or not this task is an Actor
+    #: Whether this task is an Actor
```

Comment:
```suggestion
    #: Whether this task is an Actor or a proxy object to an Actor
```
^ is that correct?

Reply:
This is a TaskState object on a Scheduler. So this object is both the actor and its proxies, depending on the worker.
More confusingly, Actor objects are the proxies, whereas the original is just whatever class the user passed to the task.